packages/compass-saved-aggregations-queries/src/utlis/queries.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { expect } from 'chai';
+import { cleanUp } from '../test/setup';
+
+import { FavoriteQueryCollection } from '@mongodb-js/compass-query-history';
```

Comment:
Looking at how slow it is to run due to the amount of code we need to import from the external library (two whole plugins, CI seems to be currently failing with a timeout because of that), the amount of code changes in other packages that are required to run this in node env, and the amount of setup we had to do here, I would maybe consider just mocking both packages instead of doing all this pre-setup. It feels to me like we are testing these external deps more than we are testing this package

Reply:
We could do that too yeah, to be fair in the new connection form we mock the connection storage (that is tested somewhere else) and tests became more stable.

Even with this approach, I don't think the reason it times out is cause it takes too long. My guess is that it fails due to something that just doesn't return though.

Reply:
This suite fails for me locally with a timeout, I did some changes to the logic to time how much the require takes in the most basic test in `index.test.ts` and it's 30s (aggregations plugin is almost 1mb packaged, query history is 600kb, both will be imported and ts-node will need to process them):

<img width="797" alt="image" src="https://user-images.githubusercontent.com/5036933/151183629-7002d2bb-78cb-4777-9380-4045c9ca4853.png">